AlignedCollectionViewFlowLayout/Classes/AlignedCollectionViewFlowLayout.swift

```diff
@@ -230,8 +233,43 @@ public class AlignedCollectionViewFlowLayout: UICollectionViewFlowLayout {
         lineFrame.size.width = lineWidth
         return super.layoutAttributesForElements(in: lineFrame) ?? []
     }
-    
-    /// Copmutes the alignment axis with which to align the items represented by the `layoutAttributes` objects vertically.
+
+    /// Determines the total width of layout attributes objects for all items including minimumLineSpacing.
+    ///
+    /// - Parameter layoutAttributes: The layout attributes for an entire line.
+    /// - Returns: The real content width for all the layout attributes objects representing all items in the same line.
+    fileprivate func contentWidthForLine(with layoutAttributes: [UICollectionViewLayoutAttributes]) -> CGFloat {
+
+        let cellLayoutAttributes = layoutAttributes.filter { $0.representedElementCategory == .cell }
+
+        let itemWidths = cellLayoutAttributes.reduce(0) { $0 + $1.frame.size.width }
+
+        let spacingWidths = minimumLineSpacing * CGFloat(cellLayoutAttributes.count-1)
+
+        return itemWidths + spacingWidths
```

Comment:
I like the use of functional methods here! 👍 

Please remove the empty lines between the code lines.